src/collective/cover/testing.py

```diff
@@ -154,27 +138,11 @@ def setUpPloneSite(self, portal):
 FIXTURE = Fixture()
 
 
-class MultipleGridsFixture(Fixture):
```

Comment:
There is no need of this fixture as this will be the default behaviour.
I merged the tests from this fixture to the Integration tests